source/about/mattermost-v10-changelog.md

```diff
@@ -7,6 +7,79 @@ Support for Mattermost Server v9.5 [Extended Support Release](https://docs.matte
 - See the [changelog in progress](https://bit.ly/2nK3cVf) for details about the upcoming release.
 ```
 
+(release-v10.4-feature-release)=
+## Release v10.4 - [Feature Release](https://docs.mattermost.com/about/release-policy.html#release-types)
+
+**Release Day: January 16, 2025**
+
+```{Important}
+If you upgrade from a release earlier than v10.3, please read the other [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html).
+```
+
+### Improvements
+
+#### User Interface (UI)
+ - Pre-packaged Calls plugin [v1.4.0](https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v1.4.0).
+ - Improved the handling of Thai script in search terms.
+ - Added tooltips to the buttons shown in the channel info right-hand side.
+ - Downgraded Spanish language to Alpha.
+ - Removed the feature to import themes from Slack.
+
+#### Administration
+ - Redis is now available as an alternative cache backend for all enterprise customers. It can be leveraged to run Mattermost at a very high scale. There is a new section in 
```

Comment:
@agnivade / @streamer45, is this ready for production use? (I'm thinking we haven't yet deployed this to Mattermost Cloud, so wondering if we should caveat the changelog accordingly.)

Reply:
We have soaked this for quite a while in Community and even fixed some bugs. This falls in the same bucket as ES/OS where we use that in Community, but not in Cloud. It should be okay for customers to start using it.